tests/main/upgrade-from-release/task.yaml

```diff
@@ -48,7 +48,7 @@ execute: |
     fi
 
     declare -A EXPECTED_SNAPD_VERSIONS=(
-        ["24.04"]='2.63.1\+24.04'
+        ["24.04"]='2.65.3\+24.04'
         ["22.04"]='2.55.3\+22.04'
```

Comment:
All of Focal, Jammy and Noble was updated today. 
See: http://archive.ubuntu.com/ubuntu/pool/main/s/snapd/

Reply:
thanks